presentation/src/main/java/com/sopt/presentation/groupCreate/groupCreateSuccess/GroupCreateSuccessRoute.kt

```diff
@@ -0,0 +1,250 @@
+                            textStyle = NoostakTheme.typography.c3Regular,
+                            textColor = NoostakTheme.colors.white,
+                            backgroundColor = NoostakTheme.colors.gray900
+                        )
+                    }
+                )
+            }
+        }
+    ) { innerPadding ->
+        Column(
+            verticalArrangement = Arrangement.SpaceBetween,
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(innerPadding)
+                .padding(horizontal = 16.dp)
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .weight(1f)
+            ) {
+                Image(
+                    painter = painterResource(id = R.drawable.ic_launcher_background),
+                    contentDescription = "Image Group Create Success",
+                    modifier = Modifier
+                        .padding(top = 56.dp)
+                        .size(108.dp)
+                        .aspectRatio(1f)
+                        .align(Alignment.CenterHorizontally)
+                )
+                Text(
+                    text = stringResource(R.string.text_group_create_success_title),
+                    color = NoostakTheme.colors.gray900,
+                    style = NoostakTheme.typography.t1SemiBold,
+                    modifier = Modifier
+                        .padding(top = 8.dp)
+                        .align(Alignment.CenterHorizontally),
+                )
+                Text(
+                    text = stringResource(R.string.text_group_create_success_content),
+                    color = NoostakTheme.colors.gray900,
+                    style = NoostakTheme.typography.b4Regular,
+                    textAlign = TextAlign.Center,
+                    modifier = Modifier
+                        .padding(top = 8.dp)
+                        .align(Alignment.CenterHorizontally),
+                )
+                Text(
+                    text = groupCode,
+                    color = NoostakTheme.colors.gray800,
+                    style = NoostakTheme.typography.codeMedium,
+                    modifier = Modifier
+                        .padding(top = 16.dp)
+                        .align(Alignment.CenterHorizontally),
+                )
+            }
+            Text(
+                text = stringResource(R.string.text_group_create_success_code_copy),
+                color = NoostakTheme.colors.gray800,
+                style = NoostakTheme.typography.c3Regular.copy(
+                    textDecoration = TextDecoration.Underline
+                ),
+                modifier = Modifier
+                    .padding(12.dp)
+                    .clickable {
+                        clipboardManager.setText(AnnotatedString(groupCode))
+                        onCopyBtnClick()
+                    }
```

Comment:
P1: noRippleClickable로 수정띠
```suggestion
                    .noRippleClickable {
                        clipboardManager.setText(AnnotatedString(groupCode))
                        onCopyBtnClick()
                    }
```

Reply:
오 ㄱㅅㄱㅅ